examples/chip-tool/commands/clusters/ModelCommand.h

```diff
@@ -82,9 +86,12 @@ class ModelCommand : public CHIPCommand
     chip::NodeId mDestinationId;
     std::vector<chip::EndpointId> mEndPointId;
     chip::Optional<bool> mIsPeerLIT;
+    chip::Optional<bool> mAllowLargePayload;
```

Comment:
since AllowLargePayload always has a value, and return false when !mAllowLargePayload.HasValue(), why do we need to make this Optional?

Reply:
It's Optional so that it doesn't have to be provided on the command line.

Reply:
That's right. the command option is not mandatory.

Reply:
I see, 'allow-large-payload'  is optional, that is why you use chip::Optional<bool> instead of bool